### PR TITLE
nodePackages: update set

### PR DESCRIPTION
**Pull request by evanjs**
###### Motivation for this change
https://github.com/angular/angular-cli/releases/tag/v9.0.0
https://github.com/angular/angular-cli/releases/tag/v9.0.1

Supercedes / closes #78151 (?)

Also, I don't see `pkgs/development/node-packages/composition-v13.nix` in #78151.  Is this something that is necessary?
___
Note that https://github.com/NixOS/nixpkgs/pull/73225 still seems to be present
Applying the patch still seems to fix the issue.
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/node-packages/node-env.nix [hunk 1]
```diff
@@ -363,7 +363,7 @@ let
 
         npm ${forceOfflineFlag} --nodedir=${nodeSources} ${npmFlags} ${stdenv.lib.optionalString production "--production"} rebuild
 
-        if [ "''${dontNpmInstall-}" != "1" ]
+        if [ "$dontNpmInstall" != "1" ]
```
Comment (Mic92) [inline pkgs/development/node-packages/node-env.nix]: Please remove this one from the PR. This should be fixed upstream in node2nix.

File: pkgs/development/node-packages/composition-v13.nix [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 {pkgs ? import <nixpkgs> {
     inherit system;
-  }, system ? builtins.currentSystem, nodejs ? pkgs."nodejs-13_x"}:
+  }, system ? builtins.currentSystem, nodejs ? pkgs."nodejs-8_x"}:
```
Comment (Mic92) [inline pkgs/development/node-packages/composition-v13.nix]: Please remove this one from the PR. This should be fixed upstream in node2nix.

File: pkgs/development/node-packages/node-env.nix (hunk 1 above)
Comment (Mic92) [reply pkgs/development/node-packages/node-env.nix]: https://github.com/svanderburg/node2nix/pull/174

Comment (evanjs) [reply pkgs/development/node-packages/node-env.nix]: Thank you! Fixed.

**Comment by Mic92** [comment]
cc @hlolli lumo is broken: https://gist.github.com/Mic92/e7ac6eb8652b72ddc9e5a349b8e7bc23

**Comment by hlolli** [comment]
does this -> https://github.com/NixOS/nixpkgs/pull/59414 fix it? @Mic92

**Comment by hlolli** [comment]
oops, it's ofc out of sync, I'll rebase and test it. Not sure if I can do it this weekend. Next week. Will update.

**Pull request merged**